package.json

```diff
@@ -19,16 +19,13 @@
     "fmt": "prettier --write 'src/*.ts'"
   },
   "dependencies": {
-    "@actions/core": "1.10.1",
-    "@actions/io": "1.1.3",
-    "@actions/tool-cache": "2.0.1",
-    "@hashicorp/github-actions-core": "github:hashicorp/github-actions-core#v0.4.1",
-    "@vercel/ncc": "0.36.1"
+    "@hashicorp/github-actions-core": "github:hashicorp/github-actions-core#1.0.0-rc1"
```

Comment:
IMO its probably worth just releasing GHA-core 1.0.0 before releasing this change